### ISSUE TITLE
Alexander

### ISSUE THREAD

**Issue by coffey55** [coffey55/Change-a-Tire#2]
Are you seeing this message?

**Comment by Alexanderpants** [comment]
Yes I can see this message. This is a good way for communication for the time being.

**Comment by Alexanderpants** [comment]
We should have a single issue for all three of us to comment in. let's create one called "group chat" or something.
